backend/endpoints/api/controller.js

```diff
@@ -1,40 +1,84 @@
 const _aufstellung = require('../aufstellungen/aufstellung');
-const _element = require('../aufstellungen/element');
 const _roles = require('../../authentication/role');
 const _termin = require('../termine/termin');
+const _db = require('../../db/connector');
+const { reduce } = require('../raids/controller');
 
 module.exports = [
-    {function: setAufstellung, path: '/aufstellung', method: 'post', authed: true},
+    {function: setAufstellung, path: '/aufstellungen', method: 'post', authed: true},
 ];
 
-
 async function setAufstellung(req, authentication) {
-    const invalidData = "{\"error\":\"invalid data\"}";
-    
     const termin = req.body.body.terminId;
     const data = req.body.body.aufstellungen;
 
+    const stmt = 'SELECT count(*) AS count FROM Termin WHERE id = ?';
+    const terminExists = await _db.queryV(stmt, [termin]);

```

Comment:
Bitte auslagern in `termine/termin.js` als `doesTerminExist` o.ä.

Reply:
erledigt als `doesTerminExist(termin)` in `termine/termin.js`